pkg/cloudprovider/providers/cloudstack/cloudstack_loadbalancer.go

```diff
@@ -23,7 +23,9 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/xanzy/go-cloudstack/cloudstack"
+
 	"k8s.io/api/core/v1"
+	cloudprovider "k8s.io/cloud-provider"
```

Comment:
@wgliang this import doesn't exist in 1.12 :(

Reply:
just realized the PR was merged for 1.13 only

Reply:
DONE